src/modules/core/components/ProgressBar/ProgressBar.tsx

```diff
@@ -42,12 +42,13 @@ const ProgressBar = ({
 }: Props) => {
   const { formatMessage } = useIntl();
   const titleText = formatMessage(MSG.titleProgress, { value, max });
+
   return (
     <div className={`${styles.wrapper} ${getMainClasses(appearance, styles)}`}>
-      {threshold && (
+      {!!threshold && (
```

Comment:
Threshold now looks good, however the questionmark icon alignment (for that line) looks wonky on my machine.

![Screenshot from 2022-07-13 21-25-43](https://user-images.githubusercontent.com/1193222/178805328-3f375968-cbb0-40b3-831e-3754a4899ed4.png)

Reply:
Same, when you get to the finalize step

![Screenshot from 2022-07-13 21-26-33](https://user-images.githubusercontent.com/1193222/178805375-28c41968-4565-4b35-847c-be5a95976711.png)

Reply:
Oh yes, these icons are annoying but I will get them 💪.

Reply:
@rdig Changes were made. Can you please review this again? Thanks.